src/components/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Footer = () => {
+    return (
+        <div className='footer'>
+            Online Store 2023
```

Comment:
Text data is better to put in some special object. E.g. APP_TEXTS.FOOTER_TITLE: "Online Store 2023"
So you can easily find it in future and replace with another text.